pkg/controller/gc/gc_controller.go

```diff
@@ -73,6 +73,7 @@ func New(kubeClient clientset.Interface, resyncPeriod controller.ResyncPeriodFun
 		&api.Pod{},
 		resyncPeriod(),
 		framework.ResourceEventHandlerFuncs{},
+		cache.Indexers{},
```

Comment:
Can you please add comment why you are not creating any indexes here? (I think it's fine in GC, but it would be good to be explicit about it).